src/css/profile/mobile/common/gridview.less

```diff
@@ -31,15 +29,15 @@ tau-gridview {
 		}
 
 		.ui-gridview-handler {
-			mask-image: url("images/core_grid_button_reorder.png");
+			mask-image: url("images/3_Controllers/tw_list_icon_reorder.svg");
 			mask-size: 100%;
 			background-color: @color_button_add;
```

Comment:
Can you change this colour value to CSS variable?

Reply:
Done.